### PR TITLE
[s2s] clean up + doc

### PR DESCRIPTION
**Pull request by stas00**
this is a follow up to https://github.com/huggingface/transformers/pull/6149
- there was no need to add newly added options to finetune.sh - reverted that change
- added a hint to users how to get all the options (--help)

@sshleifer

### CODE REVIEW

File: examples/seq2seq/README.md [hunk 1]
```diff
@@ -66,6 +66,15 @@ Summarization Tips:
 Datasets: Seq2SeqDataset will be used for all models besides MBart, for which MBartDataset will be used.**
 A new dataset is needed to support multilingual tasks.
 
+
+### Command Line Options
+
+To see all the possible command line options, run:
+
+```bash
+./finetune.sh --help
```
Comment (JetRunner) [inline examples/seq2seq/README.md:75]: why not `python finetune.py --help`? makes more sense to me.

File: examples/seq2seq/finetune.sh [hunk 2]
```diff
@@ -10,8 +11,4 @@ python finetune.py \
     --do_predict \
     --n_val 1000 \
     --val_check_interval 0.1 \
-    --encoder_layerdrop 0.1 \
-    --decoder_layerdrop 0.1 \
-    --dropout 0.1 \
-    --attention_dropout 0.1 \
```
Comment (JetRunner) [inline examples/seq2seq/finetune.sh:16]: Personally I would make `dropout = 0.1`, and other types of dropout = 0 (but still list them here). Just a suggestion. @sshleifer

File: examples/seq2seq/README.md (hunk 1 above)
Comment (stas00) [reply examples/seq2seq/README.md:75]: Either way works, but README.md uses the bash script in its usage examples:
https://github.com/huggingface/transformers/blob/master/examples/seq2seq/README.md#finetuning-training-params
Whatever you choose ideally let's make it consistent?

File: examples/seq2seq/finetune.sh (hunk 2 above)
Comment (sshleifer) [reply examples/seq2seq/finetune.sh:16]: That resulted will be produced by `BartConfig`:
```
        encoder_layerdrop=0.0,
        decoder_layerdrop=0.0,
        attention_dropout=0.0,
        dropout=0.1,
```

File: examples/seq2seq/README.md (hunk 1 above)
Comment (sshleifer) [inline examples/seq2seq/README.md:75]:
```suggestion
./finetune.sh --help  # this calls python finetune.py --help
```
